site/viamguides/src/emily-door-lock/emily-door-lock.md

```diff
@@ -0,0 +1,210 @@
+   ```json
+   {
+     "components": [
+       {
+         "name": "IR",
+         "api": "rdk:component:sensor",
+         "model": "hw201",
+         "attributes": {
+           "meta": "0 is locked, 1 is unlocked",
+           "pin": 18
+         }
+       }
+     ],
+     "services": [
+       {
+         "name": "OTA",
+         "api": "rdk:service:generic",
+         "model": "rdk:builtin:ota_service",
+         "attributes": {
+           "url": "https://github.com/ehhong/micro-rdk-with-IR-sensor/releases/download/v0.0.5/rusty-door-ota.bin",
+           "version": "v0.0.5"
+         }
+       }
+     ]
+   }
+   ```
+   ![JSON configuration](assets/jsonConfig.png)
+1. Click **Save** to apply your configuration changes. This may take a moment for the device to update.
+
+### Test the IR sensor
+
+1. Notice an IR sensor component in the left sidebar that corresponds to a panel with more details on the right. There is also an OTA service in the left sidebar that corresponds to a panel on the right.
+1. Find the IR panel, and then expand the **TEST** section to view the readings detected by the IR sensor. See what happens when you point the sensor towards the IR tape (or similar reflective surface).
+   ![getReadings](assets/getReadings.png)
+
+In the next step, let's position the detector so that a value of `0` indicates a locked state, and `1` is unlocked.
+
+<!-- ------------------------ -->
+
+## Position the detector
+
+Duration: 3
+
+Now that the detector is working as expected, let's position it near the door lock to dial in the sensitivity settings.
+
+1. Mount the device next to the door you want to monitor, pointing the sensor in the direction of the door lock.
+   ![overhead view of detector](assets/emilyOverhead.jpeg)
+1. Add IR tape (or similar reflective material) to the deadbolt.
+   ![side view of detector](assets/emilySideview.jpeg)
+1. Test it again to make sure the sensor placed in the new position captures the readings accurately and reliably. A value of `0` should indicate a locked state, and `1` is unlocked.
+
+### Remote access to door lock status
+
+Now that your sensor is set up, you may not want to log in to the Viam app every time you want to check whether your door is locked or unloacked. There's a few options to consider:
+
+- Download the Viam mobile application from the [Apple](https://apps.apple.com/us/app/viam-robotics/id6451424162) and [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US) app stores to keep track of all your Viam machines, including this door lock status detector
+- Develop your own custom mobile application using one of [Viam's SDKs](https://docs.viam.com/dev/reference/sdks/#mobile-sdk)
+- Set up a notification to alert you when you've left your home and your door is still unlocked, by [creating your own custom Viam module](https://docs.viam.com/operate/get-started/other-hardware/hello-world-module/)
```

Comment:
This guide won't really help with the micro-rdk but we could link to the "modules for ESP32" docs page which provides a pretty minimal tutorial. 

```suggestion
- Set up a notification to alert you when you've left your home and your door is still unlocked, by [creating your own custom Viam module](https://docs.viam.com/operate/get-started/other-hardware/micro-module/)
```